src/applications/settings/webui/vue-app/src/models/settings.ts

```diff
@@ -1,3 +1,5 @@
 export default class Settings {
     public deviceName: String = "";
+    public wifiSSID: String = "";
+    public wifiPassword: String = "";
```

Comment:
Please rename `wifiSSID` to `webuiWifiSSID` and `wifiPassword` to `webuiWifiPassword`.